src/script/components/Modals/PrimaryModal/PrimaryModalTypes.ts

```diff
@@ -23,6 +23,7 @@ import {ClientNotificationData} from '../../../notification/PreferenceNotificati
 
 export interface ButtonAction {
   action?: Function;
+  autoFocus?: true;
```

Comment:
I can't think of anything else, but can we find an other word?
`autoFocus` is already a React HTML attribute prop

Reply:
that guy https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L2896

Reply:
Sorry for the confusion @V-Gira, I was writing to @thisisamir98. I think the data type should be `boolean` not `true`.